src/items.rs

```diff
@@ -110,11 +111,71 @@ impl Rewrite for ast::Local {
 
         result.push_str(&infix);
 
+        let mut has_comment_between_assign_and_rhs = false;
         if let Some(ref ex) = self.init {
+            let base_span = if let Some(ref ty) = self.ty {
```

Comment:
Is it possible to move this logic inside `rewrite_assign_rhs_expr` so that we can benefit from this in other assignment-ish expressions and items?

Reply:
Thank you the review. I tried to move this logic to inside ```rewrite_assign_rhs_expr```. But I found that ```rewrite_assign_rhs_expr``` also becomes to have the role to rewrite lhs.
```rewrite_assign_rhs_expr``` is used by other methods, so I think it should not rename and rewrite lhs.

Can I revert [this commit](https://github.com/rust-lang/rustfmt/pull/3857/commits/5530066f8bbea94d7faaa091835b3965ba9e0af1)? This logic seems not to have to be moved to ```rewrite_assign_rhs_expr```.

Reply:
Thanks, in that case we can keep the current form.